edx_proctoring/backends/rest.py

```diff
@@ -76,10 +78,28 @@ def get_javascript(self):
         Returns the url of the javascript bundle into which the provider's JS will be loaded
         """
         package = self.__class__.__module__.split('.')[0]
-        bundle_chunks = get_files(package, config="WORKERS")
-        if bundle_chunks:
-            return bundle_chunks[0]["url"]
-        return ''
+        js_url = ''
+        try:
+            bundle_chunks = get_files(package, config="WORKERS")
```

Comment:
Is it still possible for this method to raise IOError? I know the tests still have a case expecting to catch IOError, but this method should probably be resilient enough to handle misconfiguration or missing npm packages, etc. (as long as the warning messages are clear for each edge case)